core/src/main/scala/org/apache/spark/sql/delta/commands/cdc/CDCReader.scala

```diff
@@ -773,7 +773,7 @@ trait CDCReaderImpl extends DeltaLogging {
             deltaLog.dataPath,
             snapshot,
             rowIndexFilters =
-              Some(fileActionsToRowIndexFilters(addFiles.toSeq, useIfContainedFilters = true))),
+              Some(fileActionsToIfNotContainedRowIndexFilters(addFiles.toSeq))),
```

Comment:
nit: There is a typo inL763 and L782: generateFileActionWithInlineDv -> generateFileActionsWithInlineDv